mmdet3d/models/segmentors/encoder_decoder.py

```diff
@@ -130,6 +142,19 @@ def _auxiliary_head_forward_train(self, x, img_metas, pts_semantic_mask):
 
         return losses
 
+    def _regularization_loss_forward_train(self):
+        """Calculate regularization loss for model weight in training."""
+        losses = dict()
+        if isinstance(self.regularization_loss, nn.ModuleList):
+            for idx, reg_loss in enumerate(self.regularization_loss):
```

Comment:
reg_loss will cause ambiguity as we have `reg_loss` for regression.

Reply:
Fix to regularize_loss.